src/Configuration/Defaults/CommonRules.php

```diff
@@ -304,5 +305,6 @@ class CommonRules extends Rules
         LineEndingFixer::class => true,
         StatementIndentationFixer::class => true,
         BlankLineBetweenImportGroupsFixer::class => true,
+        BlankLineBeforeStatementFixer::class => true,
```

Comment:
According to documentation, this will add empty line not only before `return`. 
Default config:
```        
    'break',
    'continue',
    'declare',
    'return',
    'throw',
    'try',
]
```
https://mlocati.github.io/php-cs-fixer-configurator/#version:3.16|fixer:blank_line_before_statement

Reply:
I would accept it and even add more statements (if, loops)

Reply:
Added more statements and cases for test.